food-truck-frontend/src/components/recommended-trucks/SearchRecommendedTrucksForm.tsx

```diff
@@ -1,18 +1,17 @@
-import { LatLng } from "@google/maps";
-import { Button, Slider, TextField, Typography } from "@material-ui/core";
-import React, { ChangeEvent, Component } from "react";
-import { DEFAULT_ERR_KICK, DEFAULT_ERR_RESP } from "../../api/DefaultResponses";
+import {LatLng} from "@google/maps";
+import {Button, Slider, TextField, Typography} from "@material-ui/core";
+import React, {ChangeEvent, Component} from "react";
+import {DEFAULT_ERR_RESP} from "../../api/DefaultResponses";
 import api from "../../util/api";
-import Form from "../Form";
-import { MoneyRating } from "../truck/rate_and_review/ratings";
+import {MoneyRating} from "../truck/rate_and_review/ratings";
 import MultiField from "./multi_field";
 
 type RecommendedTruckProps = {};
 type RecommendedTruckState = {
-  location: LatLng;
-  acceptibleRadius: number;
-  priceRating: number;
-  foodCategory: string;
+    location: LatLng;
+    acceptibleRadius: number;
+    priceRating: number;
+    foodCategory: string;
   menuItems: string[];
```

Comment:
Indent inconsistency?

Reply:
Oh yeah. My formatter has different ideas of how things should be indented (since I've been using Prettier in VSCode, my default), so if I accidentally auto-format, it'll mess with indents for no reason. I try to avoid letting it happen, but sometimes my Ctrl+Shift+F instincts kick in.